Description: render build reproducible
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-19

--- a/scripts/release/build.ts
+++ b/scripts/release/build.ts
@@ -31,7 +31,7 @@
  */
 export const generateMeta = (): any => ({
   version: packageJson.version,
-  timestamp: new Date(),
+  timestamp: new Date(process.env.SOURCE_DATE_EPOCH ? (parseInt(process.env.SOURCE_DATE_EPOCH) * 1000) : new Date().getTime()),
 });
 
 /**
